ext/rotoscope/rotoscope.c

```diff
@@ -188,7 +221,18 @@ static void event_hook(VALUE tpval, void *data)
     return;
 
   rs_tracepoint_t trace = extract_full_tracevals(trace_arg, &trace_path);
-  fprintf(config->log, RS_CSV_FORMAT, RS_CSV_VALUES(trace));
+  if (!strcmp("Rotoscope", trace.entity))
+    return;
```

Comment:
I noticed in writing this PR that we were writing "dangling" events for `Rotoscope`, e.g. `CALL Rotoscope#stop_trace` without the `RETURN` event being captured, obviously due to our script not running when the paired event is fired. This line to ignore all `Rotoscope` calls is the easy way out.